internal/errors/errors.go

```diff
@@ -5,17 +5,28 @@ import (
 	"errors"
 	"fmt"
 	"io"
+
+	grpcCodes "google.golang.org/grpc/codes"
+
+	"github.com/ydb-platform/ydb-go-genproto/protos/Ydb"
 )
 
+type Error interface {
+	error
+
+	Code() int32
```

Comment:
suppose it'll be better to leave this as wrapper for op error, while transport error should always be represented by status package